record-boot-success/record-boot-success.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Record Boot Success
+ConditionFileIsExecutable=/usr/bin/grub-editenv
```

Comment:
Have you tested what happens if this condition fails? Do we get errors in the journal from `record-boot-success.desktop`? I guess `StartServiceByName` will fail and so the `dbus-send` incantation will fail. Does this have adverse consequences?